src/main/scala/io/pravega/connectors/spark/PravegaSourceProvider.scala

```diff
@@ -201,6 +202,12 @@ object PravegaSourceProvider extends Logging {
     if (retentionMilliseconds.isDefined && (Try(retentionMilliseconds.get.toInt).isFailure || retentionMilliseconds.get.toInt < 0)) {
       throw new IllegalArgumentException(s"Retention time should be an integer morethan or equal to zero milliseconds")
     }
+
+    val maxOffsetPerTrigger = caseInsensitiveParams.get(PravegaSourceProvider.MAX_OFFSET_PER_TRIGGER)
+    if (maxOffsetPerTrigger.isDefined && (Try(maxOffsetPerTrigger.get.toInt).isFailure || maxOffsetPerTrigger.get.toInt < 1))
+    {
+      throw new IllegalArgumentException(s"Max events per trigger should be an integer more than or equal to one")

```

Comment:
Max offset per trigger should be of type Long, right? not integer

Reply:
done